packages/lesswrong/unitTests/sql/UpdateQuery.tests.ts

```diff
@@ -82,10 +82,10 @@ describe("UpdateQuery", () => {
       expectedArgs: [4, 3],
     },
     {
-      name: "can add a correct type hint for native arrays of JSON",
+      name: "can add a correct type hint for native arrays of JSON in a JSONB (rather than JSONB[]) field",
       getQuery: () => new UpdateQuery<DbTestObject>(testTable, {a: 3}, {$set: {c: [{d: "test"}]}}),
-      expectedSql: `UPDATE "TestCollection" SET "c" = $1::JSONB[] WHERE "a" = $2 RETURNING "_id"`,
-      expectedArgs: [[{d: "test"}], 3],
+      expectedSql: `UPDATE "TestCollection" SET "c" = $1::JSONB WHERE "a" = $2 RETURNING "_id"`,
```

Comment:
I believe this was explicitly testing for behavior that would break if you tried to run the SQL (i.e. this is what was causing the `error: column "rawFeed" is of type jsonb but expression is of type jsonb[]` error in the rss-integration cron job).  However, I'm _not sure_; if this is in fact correct in some circumstances then the fix probably needs to be different.